tdrs-backend/tdpservice/users/api/authorization_check.py

```diff
@@ -66,6 +66,8 @@ def get(self, request, *args, **kwargs):
         user_in_valid_group = user.is_ofa_sys_admin
 
         if (user.hhs_id is not None and user_in_valid_group) or settings.BYPASS_KIBANA_AUTH:
-            return HttpResponseRedirect(settings.KIBANA_BASE_URL)
+            logger.debug(f"User: {user} has correct authentication credentials. Allowing access to Kibana.")
```

Comment:
noticing this line of code trigger 8-10x in the logs after clicking kibana endpoint.

Reply:
This log line will be hit every time the frontend makes a request to Kibana. This is because we need to pass through the authentication view every time prior to proxying the request to Kibana to ensure the user is still passing the traditional Auth Check and their group hasnt changed. Debug is the highest level for the logger so it wont show up in prod.